data/organizations.json

```diff
@@ -40,5 +40,11 @@
     "Meaning": "StackOverflow",
     "URL": "https://stackoverflow.com",
     "What they do": "Popular question and answer website"
-  }
-]
\ No newline at end of file
+  },
+  {
+    "Abbreviation": "YT",
+    "Meaning": "VIDEO SHARING AND STREAMING PLATFORM",
```

Comment:
Could you change the meaning to be just "YouTube"? Thank you for your contirbution :+1: